app/javascript/dashboard/mixins/agentMixin.js

```diff
@@ -0,0 +1,35 @@
+import { mapGetters } from 'vuex';
+
+export default {
+  computed: {
+    getAgents() {
```

Comment:
As this is a computed property let us not try to use a variable name that resembles the method name. I guess the `get` or `check` prefixes is suited for methods rather than a computed property. A suggestion here is to name this property `assignableAgents`